lib/reverse_string.rb

```diff
@@ -1,5 +1,17 @@
 def reverse_characters(str)
+    newarr = []
+	str = str.split("")
+	str.length.times do
+		newarr.push(str.pop)
+		end
```

Comment:
Very nice work! Typically, we indent the line that ends a block the same amount as the beginning line, so line 6 should be in line with line 4.